### ISSUE TITLE
to fix before 2.6.0 release

### ISSUE THREAD

**Issue by rienafairefr** [iot-lab/cli-tools#45]
IMO, 2.6.0 might not be ready for release:

- warnings: when `iotlab ssh` can be run (if ssh-cli-tools is installed), then we get this mouthful on top of every invocation 
```
/home/matthieu/.virtualenvs/cli-tools/local/lib/python2.7/site-packages/pssh/clients/miko/parallel.py:48: UserWarning: This client will be replaced as the default client by the better performing and non-monkey patching pssh.clients.native.ParallelSSHClient from 2.0.0 onwards.
Please ensure required functionality is supported by the new client by switching to 'from pssh.clients import ParallelSSHClient'. The pssh2_client import for the new client will continue to be supported for backwards compatibility. 

To continue using this client please update imports to 'from pssh.clients.miko import ParallelSSHClient'.
  warn(_msg)
```

- `iotlab` subcommand does not show the correct program in usage:

```
(cli-tools) matthieu@zurkus:~/iot-lab-dev/parts/cli-tools (doc)$ iotlab node
usage: iotlab [-h] [-u USERNAME] [-p PASSWORD] [-v] [--jmespath JMESPATH]
              [--format FORMAT] [-i EXPERIMENT_ID]
              (-sta | -sto | -r | --update-idle | --debug-start | --debug-stop | -up FIRMWARE_PATH | --profile PROFILE_NAME | --profile-load PROFILE_PATH | --profile-reset)
              [-e EXCLUDE_NODES_LIST | -l NODES_LIST]
```
`iotlab` should be `iotlab node` here

**Comment by aabadie** [comment]
The first problem is more related to ssh-cli-tools. Maybe https://github.com/iot-lab/ssh-cli-tools/pull/40 could help.

I agree that the second point if a problem.

**Comment by rienafairefr** [comment]
OK, for the first problem I agree. The second problem I tried to fix with #46
